src/charm.py

```diff
@@ -330,7 +330,6 @@ def configure_ceph(self) -> None:
         """
         cmds = [
             ["ceph", "config", "set", "global", "mon_allow_pool_size_one", "true"],
```

Comment:
should this line be removed as well ?

Reply:
or rather, do we even need the `configure_ceph` method ?

Reply:
I think it can be useful down the road. The reason I didn't remove the other command is because it's harmless (and we may need it in case we support single node clusters in the charm).

Reply:
hmm, It is harmless but MicroCeph supports single node Ceph clusters (with replication enabled for sustaining OSD failures.). The use-case which might benefit from the this config would be `Single Node - Single OSD` for cases where 3 block devices might not be available effectively disabling replication. For that, we do have Loop OSD support in the charm. Hence, I believe `mon_allow_pool_size_one` does not help any use-case. We may leave it as is but removing it feels cleaner to me.

Reply:
Fair enough. I'll remove it.